src/components/Calculator.js

```diff
@@ -1,33 +1,55 @@
 import React from 'react';
 import '../index.css';
+import calculate from '../logic/calculate';
 
```

Comment:
- [ ] For this milestone, we are expecting you to practice what you have learned until now; like passing props to child components, using lifecycle methods, lifting state up. 

Tip:
_You should divide your `Calculator` component into child components in order to practice these skills. (For example, you can create a display component for displaying calculations and a keypad component for the keys.)_


>Practice what you have learned until now: use state and props, pass props to child components, use lifecycle methods, lift the state up.

Reply:
Hello @sinansevgi. Thanks for the insightful review. I have implemented the required changes as we speak. Hopefully, my project gets approved. :smiley: 
Happy Codding... :computer: